contrib/splitup.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -xe
+
+# VARIABLES
+BASEDIR=$(dirname "$0")
+LOCAL_BRANCH=$(cd $BASEDIR && git rev-parse --abbrev-ref HEAD)
+
+
+# FUNCTIONS
+function check_existing_remote {
+  if ! git remote show $1 &> /dev/null; then
```

Comment:
Nevermind I was trying with a repo that doesn't exist:
+ check_existing_remote ceph-test
+ git remote show ceph-test
+ git remote add ceph-test git@github.com:/ceph/ansible-ceph-test.git
+ check_existing_remote ceph-test
+ git remote show ceph-test
ERROR: Repository not found.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
+ git remote add ceph-test git@github.com:/ceph/ansible-ceph-test.git
fatal: remote ceph-test already exists.

It works fine if the repo exists :)